src/components/IGVBrowser.js

```diff
@@ -88,7 +88,7 @@ const IGVBrowser = ({ selectedFiles, refGenome: { genome, reference }, workspace
       const simpleUrl = _.last(url.split('/')).split('?')[0];
 
       const fullUrl = isSignedUrl ? url : Utils.mergeQueryParams(userProjectParam, url);
-      const fullIndexUrl = isSignedUrl ? indexURL : Utils.mergeQueryParams(userProjectParam, indexURL);
+      const fullIndexUrl = isSignedUrl ? indexURL : indexURL && Utils.mergeQueryParams(userProjectParam, indexURL);
```

Comment:
I realize you tested this and that it works in practice. For my curiosity: why is this needed for `indexURL` but not for  `url`? Is it because if a user is adding a track, `url` will always be valid?